### ISSUE TITLE
Cant use more than two prompts

### ISSUE THREAD

**Issue by pkaczynski** [bionic-gpt/bionic-gpt#410]
### Describe your issue

There is a strange error 422 when chatting (no response is shown on the chat screen) and the prompt is empty

```
2024-03-19T17:51:41.556109Z ERROR axum_server::errors: response="status = 422 Unprocessable Entity, message = query returned an unexpected number of rows"
```

### Describe your setup/startup scripts

I'm using default docker-compose and servin ollama from an official ollama docker.

### Steps to reproduce

1. Changed llama2 model to point to proper url `http://x.x.x.x:11434/vi`
2. Test execute chat - ok
3. Add another prompt that uses the same model
4. test execute chat using the second prompt - works. Execute using the first prompt (default) - no answer
5. Add another prompt using the same model
6. Test execute chat - nothing works
7. Delete third prompt
8. Seciond prompt works back again

### Any additional information

Seems like this is an app issue since even disabling ollama does not change the behaviour. How can I enable more DEBUG in axiom server to see what this unprocessable entity contains?

**Comment by pkaczynski** [comment]
<img width="1170" alt="Zrzut ekranu 2024-03-19 191005" src="https://github.com/bionic-gpt/bionic-gpt/assets/3531414/791f043b-3aff-4400-b335-46d83dfe735d">

**Comment by PatrickChudalla** [comment]
We have experienced a very similar behaviour. We had configured multiple prompts with multiple models hosted in the ollama service.
It worked for a while - well, until it didnt. We also got an empty prompt while trying to communication with the model via curls still works. 

`curl http://<IP-Address>:11434/api/generate -d '{
  "model": "vicuna:7b-16k",
  "prompt":"Why is the sky blue?"
}'`

And we got following error message too

> "status = 422 Unprocessable Entity, message = query returned an unexpected number of rows"

We had to remove all but one prompt and then it worked again. Now whenever we add one more prompt it stops working. This becomes very frustrating as we try to work with different LLM models to figure out, which one suits us best.
